bus-mapping/src/evm/opcodes/ids.rs

```diff
@@ -486,149 +487,149 @@ impl FromStr for OpcodeId {
 
     fn from_str(s: &str) -> Result<Self, Self::Err> {
```

Comment:
I suggest change `from_str("0x00u8")` into a new API called `from_u8` and keep the `from_str` as is.

Reply:
Has been adjusted.